cfme/base/login.py

```diff
@@ -15,9 +15,9 @@ class BaseLoggedInPage(View):
     """
     CSRF_TOKEN = '//meta[@name="csrf-token"]'
     flash = FlashMessages('.//div[@id="flash_msg_div"]')
-    help = NavDropdown('.//li[./a[@id="dropdownMenu1"]]|.//li[./a[@id="help-menu"]]')
-    configuration_settings = Text('.//li[./a[@title="Configuration"]]')      # 5.11+
-    settings = SettingsNavDropdown('.//li[./a[@id="dropdownMenu2"]]')
+    help = NavDropdown(id="help-menu")
+    configuration_settings = Text('//li[.//a[@title="Configuration"]]')  # 5.11+
```

Comment:
I see this is not new but what does it mean by `#5.11+` ? What happens on other stream?

Reply:
This widget doesn't exist in 5.10, configuration is accessed through the settings widget.